python/function.py

```diff
@@ -592,6 +644,14 @@ def not_in_set_of_values(self, values):
 
 	@classmethod
 	def lookup_table_value(self, lookup_table, mapping):
+		"""
+		Create a PossibleValueSet object for a value which is a member of a 
+		lookuptable.
+
+		:param list(LookupTableEntry) lookup_table: List of table entries
+		:param dict of (int, int) mapping: Mapping used for resolution
```

Comment:
Did you double check this is the pydoc formatting for the dict? I guess it doesn't matter too much since we'll move to proper type annotations soon.

Reply:
I tried to follow the format as was written [https://github.com/Vector35/binaryninja-api/blob/dev/python/binaryview.py#L3662](https://github.com/Vector35/binaryninja-api/blob/dev/python/binaryview.py#L3662). Validated - https://stackoverflow.com/questions/26712533/formatting-python-docstrings-for-dicts